### PR TITLE
Refactor: use LookPath instead of custom solution (Endpoint.IsAvailable)

### PR DESCRIPTION
**Pull request by gerbsec**

### CODE REVIEW

**Comment by gerbsec** [comment]
tested, looks good

File: endpoint/endpoint.go [hunk 1]
```diff
@@ -29,11 +29,11 @@ func Stop(code int) {
 }
 
 func RunWithTimeout(function fn) {
-    go function()
-    select {
-    case <- time.After(10 * time.Second):
-        os.Exit(102)
-    }
+	go function()
```
Comment (bfuzzy1) [inline endpoint/endpoint.go:32]: Fix the formatting here please. Is there a way to stop this from happening again?

**Review by bfuzzy1 — CHANGES_REQUESTED** [review]

File: endpoint/endpoint.go (hunk 1 above)
Comment (gerbsec) [reply endpoint/endpoint.go:32]: yeah i can undo that change buts its the only part of the code that uses spaces instead of tabs... cause mahina pushed it

Comment (bfuzzy1) [reply endpoint/endpoint.go:32]: yes please undo and submit a format fix pr.

**Review by bfuzzy1 — APPROVED** [review]

**Review by privateducky — APPROVED** [review]

**Pull request merged**